pgcli/packages/pgspecial/main.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from collections import namedtuple
+
+from . import export
+
+log = logging.getLogger(__name__)
+
+NO_QUERY = 0
+PARSED_QUERY = 1
+RAW_QUERY = 2
+
+SpecialCommand = namedtuple('SpecialCommand',
+        ['handler', 'syntax', 'description', 'arg_type', 'hidden', 'case_sensitive'])
+
+COMMANDS = {}
+
+@export
+def parse_special_command(sql):
+    command, _, arg = sql.partition(' ')
+    verbose = '+' in command
+
+    command = command.strip().replace('+', '')
+    return (command, verbose, arg.strip())
+
+def special_command(command, syntax, description, arg_type=PARSED_QUERY,
+        hidden=False, case_sensitive=True):
+    def wrapper(wrapped):
+        register_special_command(wrapped, command, syntax, description,
+                arg_type, hidden, case_sensitive)
+        return wrapped
+    return wrapper
+
+@export
+def register_special_command(handler, command, syntax, description,
+        arg_type=PARSED_QUERY, hidden=False, case_sensitive=True):
+    cmd = command.lower() if not case_sensitive else command
+    COMMANDS[cmd] = SpecialCommand(handler, syntax, description, arg_type,
+                                   hidden, case_sensitive)
+
+@export
+def execute(cur, sql):
+    command, verbose, pattern = parse_special_command(sql)
```

Comment:
can you move this line under `elif special_cmd.arg_type == PARSED_QUERY`? It's the only branch that uses it